synapse/app/_base.py

```diff
@@ -429,7 +429,7 @@ def listen_http(
         path = listener_config.path
         mode = listener_config.mode
         ports = listen_unix(path, mode, site, reactor=reactor)
-        logger.info(f"Synapse now listening on Unix Socket at: {ports[0]}")
+        logger.info(f"Synapse now listening on Unix Socket at: {ports[0].port}")
```

Comment:
it's a unix socket, it doesn't have a port (??)

Reply:
I changed up how this looks, hopefully it's a bit better now.
Fixed up in [547ff5f](https://github.com/matrix-org/synapse/pull/15353/commits/547ff5fdd4649f782eecf4189d3a2c7c7ee753cf)